sno/working_copy/postgis.py

```diff
@@ -409,27 +339,21 @@ def write_meta_crs(self, dataset):
             return
 
         with self.session() as db:
-            dbcur = db.cursor()
-            for row in spatial_refs:
-                # We do not automatically overwrite a CRS if it seems likely to be one
-                # of the Postgis builtin definitions - Postgis has lots of EPSG and ESRI
-                # definitions built-in, plus the 900913 (GOOGLE) definition.
-                # See POSTGIS_WC.md for help on working with CRS definitions in a Postgis WC.
-                dbcur.execute(
-                    SQL(
-                        """
-                        INSERT INTO public.spatial_ref_sys AS SRS ({}) VALUES ({})
-                        ON CONFLICT (srid) DO UPDATE
-                            SET (auth_name, auth_srid, srtext, proj4text)
-                            = (EXCLUDED.auth_name, EXCLUDED.auth_srid, EXCLUDED.srtext, EXCLUDED.proj4text)
-                            WHERE SRS.auth_name NOT IN ('EPSG', 'ESRI') AND SRS.srid <> 900913;
-                        """
-                    ).format(
-                        SQL(",").join([Identifier(k) for k in row]),
-                        SQL(",").join([SQL("%s")] * len(row)),
-                    ),
-                    tuple(row.values()),
-                )
+            # We do not automatically overwrite a CRS if it seems likely to be one
+            # of the Postgis builtin definitions - Postgis has lots of EPSG and ESRI
+            # definitions built-in, plus the 900913 (GOOGLE) definition.
+            # See POSTGIS_WC.md for help on working with CRS definitions in a Postgis WC.
+            db.execute(
+                """
+                INSERT INTO public.spatial_ref_sys AS SRS (srid, auth_name, auth_srid, srtext, proj4text)
```

Comment:
`spatial_ref_sys` isn't necessarily in `public`... it can be installed into any schema.

That said, if someone does move it into a `postgis` schema or wherever, it's IMO up to them to make sure it's in the db/user `search_path` — so we can always access it _without_ a schema qualifier. Same applies to `ST_Something()` functions/etc.

Reply:
Ok, I'll remove the public from all PostGIS metadata-table identifiers.